src/locales/en/translation.json

```diff
@@ -489,27 +489,27 @@
       "header": "Enter your new mobile phone number",
       "info": {
         "paragraph1": "We will send a 6 digit security code to the number you give us.",
-        "paragraph2": "You must use a UK mobile phone number.",
         "backLink": "I do not want to change my phone number"
       },
       "ukPhoneNumber": {
         "label": "UK mobile phone number",
         "validationError": {
-          "required": "Enter a UK phone number",
-          "international": "Enter a UK phone number",
-          "length": "Enter a UK phone number, like 07700 900000",
+          "required": "Enter a UK mobile phone number",
+          "international": "Enter a UK mobile phone number",
+          "length": "Enter a UK mobile phone number, like 07700 900000",
           "plusNumericOnly": "Enter a UK mobile phone number using only numbers or the + symbol",
-          "samePhoneNumber": "You are already using that phone number. Enter a different phone number."
+          "samePhoneNumber": "You're already using that phone number. Enter a different phone number"
```

Comment:
My only suggestion in this PR would be to use "smart" curly quotes (or apostrophes or whatever they are called) instead of the straight ones in the new content we are adding. Might be good to confirm with Ale or someone else from the content squad, but I'm pretty sure the curly style is more typographically correct punctuation.